server/controllers/user.controller.js

```diff
@@ -24,11 +24,19 @@ export function createUser(req, res, next) {
           if (loginErr) {
             return next(loginErr);
           }
-          res.json({
-            email: req.user.email,
-            username: req.user.username,
-            preferences: req.user.preferences,
-            id: req.user._id
+          mail.send('email-verification', {
+            body: {
+              link: `http://${req.headers.host}/api/verify?t=${auth.createVerificationToken(req.body.email)}`
```

Comment:
@catarak Do you want me to redirect `/verify` to `/api/verify' in routes/server.routes.js?